src/components/Carousel/Carousel.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useMemo, useCallback } from 'react';
+import CarouselTrack from './CarouselTrack';
+import styles from './Carousel.module.css';
+
+export default function Carousel(props) {
+    const { 
+        itemsPerPage,
+        children, 
+    } = props;
+
+    const _itemsPerPage = itemsPerPage || 1;
+    const childrenWithKeys = useMemo(() => {
+        return React.Children.toArray(children);
+    }, [children]);
+    const [index, setIndex] = useState(0);
+    const handleOnLeftClick = useCallback(() => {
+        setIndex((index) => index === 0 ? index : index - 1);
+    }, [setIndex]);
+    const handleOnRightClick = useCallback(() => {
+        setIndex((index) => index === childrenWithKeys.length - 1 ? index : index + 1);
+    }, [setIndex]);
+
+    return (
+        <div className={styles['carousel']}>
+            <CarouselTrack
+                index={index}
+                itemsPerPage={_itemsPerPage}
+            >
+                {childrenWithKeys}
+            </CarouselTrack>
+            <div className={styles['carousel__buttons']}>
+                <div className={styles['carousel__buttons-container']}>
+                    <div className={styles['carousel__direction-buttons-container']}>
+                        <button style={{backgroundColor: 'blue'}} 
+                            onClick={handleOnLeftClick}
+                        />
+                        <button style={{backgroundColor: 'blue'}}
+                            onClick={handleOnRightClick}
+                        />
+                    </div>
+                    <button className={styles['carousel__eye-button']} style={{backgroundColor: '#5CD568'}}/>
+                    <button className={styles['carousel__download-button']} style={{backgroundColor: '#33C9AD'}}/>
```

Comment:
adhere to colours specified in https://user-images.githubusercontent.com/64981786/82742743-d8877080-9d2f-11ea-9a88-5a6740b22ae9.png

Reply:
At the moment I'm not using the correct colours because I think we're using images?

Reply:
the carousel images are images, but the buttons are buttons and thus should adhere to the colour scheme? I'm confused what you're saying